### DISCUSSION TITLE
组件更新频率及维护人员安排

### DISCUSSION THREAD

**Discussion by wangdaodao** [ElemeFE/element#22888] [General]
目前老的项目还在用vue2+element，但是组件维护及更新频率太慢，上次版本更新到现在差不多9个月了，能否固定一下更新频率及维护人员，保证组件能够持续迭代。
另外组件主题工具反馈一直无法使用，这块如果不维护了，可否把源码都开源出来，社区来部署维护？

**Comment by webvs2** [comment]
What version is currently used by the project

**Comment by NidhoggDJoking** [comment]
[多主题组件预览已经很长时间无法使用了](https://element.eleme.cn/#/zh-CN/theme/preview)

https://element-api.ele.me/element/theme/getVariable?version=2.15.14
https://element-api.ele.me/element/theme/updateVariable?version=2.15.14

是这2个请求失败造成的？
是否可以派人解决

**Comment by janily** [comment]
这个是可以开发出来给社区来维护，或者源码放出来，开发者本地来使用也可以

**Reply by webvs2** [reply]
https://github.com/webvs2/element-ui-ce
可以关注这个社区版本，提交issues后将提供修复

**Comment by janily** [comment]
我想使用主题工具，可否发一份

王叨叨 ***@***.***> 于2024年11月20日周三 22:29写道：

**Comment by zuibunan** [comment]
@iamkun

**Comment by 1437647291** [comment]
element-ui的主题色可以使用css变量进行覆盖吗？

**Comment by webvs2** [comment]
请使用 element-ui-ce社区版本来修复问题，后续问题大概率会在这个社区版本上继续维护
https://www.npmjs.com/package/element-ui-ce
